users/vieting/experiments/switchboard/ctc/feat/conformer_baseline.py

```diff
@@ -154,6 +155,25 @@ def run_test_mel():
                 },
                 report_args={"architecture": "conf-wei", "lr": "wei_peak_4e-4", "specaug": "wei_adapt_80dim"},
             ),
+            "lgm80_conf-audio_perturbation": dict(
+                returnn_args={
+                    "conformer_type": "wei",
```

Comment:
```suggestion
                    "conformer_type": "wei",
                    "specaug_old": {},
```
This was better so far.